### ISSUE TITLE
Swift 2.0 Support

### ISSUE THREAD

**Issue by yankodimitrov** [yankodimitrov/SwiftPasscodeLock#14]

**Comment by gblotter** [comment]
Would you be open to having these changes either in a new branch, or with a 2.0 tag?  I've made updates to the Swift code to work with the new 2.0 beta 6, that I could make a pull request for... Up to you!

**Comment by ziogaschr** [comment]
I have added some more stuff in you old code (master branch). I will not create a PR for those now, as I plan to add them later to the new code you are working on (swift-2.0 branch).

I am adding this comment here as you might be interested in adding the features that I worked on in a different way in you new code. So feel free to take a look "[Add Touch ID button](https://github.com/ziogaschr/SwiftPasscodeLock/tree/add-touchid-button)" and "[Add passcode expiry](https://github.com/ziogaschr/SwiftPasscodeLock/tree/add-expiry-time)".

p.s: I haven't added example use in your project or any tests as I will have to do that in the new code. Let me know if you want me to add an example use or any notes here.
